.travis.yml

```diff
@@ -30,10 +30,8 @@ before_install:
 
 # Install packages
 install:
-  - conda install --yes python=$TRAVIS_PYTHON_VERSION --file requirements.txt
-  - conda install --yes python=$TRAVIS_PYTHON_VERSION jupyter
-  - conda install --yes python=$TRAVIS_PYTHON_VERSION pyflakes
-  - conda install --yes python=$TRAVIS_PYTHON_VERSION pep8
+  - conda install --yes python=$TRAVIS_PYTHON_VERSION scipy matplotlib h5py jupyter
```

Comment:
travis-world: in travis we do not install the package but we test source-branches. in that case, we can just stay with the description as it was.

if we want to *additionally* test the on-demand build packages with *each commit* then we should extent the build matrix to that use case. I did so in [Forthon](https://github.com/dpgrote/Forthon/blob/8eb122f031adc17c4f6f925d249badcc2a718ccf/.travis.yml#L26-L29) and I would like to do that in an independent PR because it will be a bit more, independent code.